src/OnboardingSPA/pages/Steps/DesignColors/index.js

```diff
@@ -50,7 +54,10 @@ const StepDesignColors = () => {
 	};
 
 	useEffect( () => {
-		if ( THEME_STATUS_ACTIVE === themeStatus ) getStylesAndPatterns();
+		if ( THEME_STATUS_ACTIVE === themeStatus ) {
+			flushQueue();
```

Comment:
If we could somehow await this and check the queue if it is empty before we make a request won't that work?